### ISSUE TITLE
QubesOS doesn't start on V560TU

### ISSUE THREAD

**Issue by filipleple** [Dasharo/dasharo-issues#903]
### Component

Dasharo firmware

### Device

NovaCustom V56 14th Gen

### Dasharo version

v0.9.0-rc2

### Brief summary

QubesOS installed with the default settings, default kernel (`6.6.21-1.qubes.fc37.x86_64`) hangs at the loading screen at nearly 100%

### How reproducible

100%

### How to reproduce

Install QubesOS with default settings and try to boot it

### Expected behavior

Login prompt after ~5min

### Actual behavior

Login prompt doesn't appear after over 5min, presumably will never appear

**Comment by wessel-novacustom** [comment]
Probably needs integration for MTL platforms. @marmarek, will it be in the next Qubes Release Candidate (rc)?

**Comment by marmarek** [comment]
Which kernel was chosen during installation? The "latest" one or the default one?

While I don't see this specific issue on my system (I've chosen "latest" kernel, which is 6.9.x), but there are others.
One already identified is lack of FLR on the ethernet controller, which prevents sys-net from starting. It isn't clear yet what can be a solution, and whether it can be solved in software at all (some more discussion on it on Matrix).

**Comment by BeataZdunczyk** [comment]
> Which kernel was chosen during installation?

@filipleple could you please provide more details

**Comment by filipleple** [comment]
@marmarek @BeataZdunczyk It was the default one, `6.6.21-1.qubes.fc37.x86_64`. Will try with the latest one in the next RC binary testing phase.

**Comment by filipleple** [comment]
So far, I can confirm Qubes does boot on **V54** with the latest kernel.

**Comment by heheihahthe** [comment]
I was able to start sys-net on the V54 with the latest kernel after following the steps in [this post](https://forum.qubes-os.org/t/sys-net-not-starting-because-it-is-unable-to-reset-pci-device/15056/9) (ignoring bios settings). Is there any reason this configuration might be undesirable?

**Comment by marmarek** [comment]
> Is there any reason this configuration might be undesirable?

Yes, restarting sys-net doesn't reset network device anymore, which weakens its isolation. This device seems to have functional FLR (function level reset - one of the reset methods), but it has a buggy advertising this feature. We're working on a workaround...

**Comment by CoinCatalysta** [comment]
> > Is there any reason this configuration might be undesirable?
> 
> Yes, restarting sys-net doesn't reset network device anymore, which weakens its isolation. This device seems to have functional FLR (function level reset - one of the reset methods), but it has a buggy advertising this feature. We're working on a workaround...

Any updates on the feasibility of solving the buggy FLR advertising? Thinking about buying a NovaCustom laptop and this issue is the main blocker for me. Thank you!

**Comment by wessel-novacustom** [comment]
> > > Is there any reason this configuration might be undesirable?
> > 
> > 
> > Yes, restarting sys-net doesn't reset network device anymore, which weakens its isolation. This device seems to have functional FLR (function level reset - one of the reset methods), but it has a buggy advertising this feature. We're working on a workaround...
> 
> Any updates on the feasibility of solving the buggy FLR advertising? Thinking about buying a NovaCustom laptop and this issue is the main blocker for me. Thank you!

@CoinCatalysta We simply need to wait for Qubes OS R4.3.

**Comment by marmarek** [comment]
> Any updates on the feasibility of solving the buggy FLR advertising? Thinking about buying a NovaCustom laptop and this issue is the main blocker for me. Thank you!

https://github.com/QubesOS/qubes-issues/issues/9356 patch is applied and it's included in 4.2 [weekly builds already](https://qubes.notset.fr/iso-testing/). When we have fixes for all the other issues (which may or may not need changes in qubes) we will release R4.2 point release (4.2.3).

**Comment by pietrushnic** [comment]
@marmarek, okay, it looks like I faced this issue. Would you like me to use weekly 4.2 or 4.2.3-rc1? RC-1 warns during installation that it is not for production workloads, but for that hardware, there seems to be no other option.

**Comment by marmarek** [comment]
Yes, get 4.2.3-rc1. Or wait for final 4.2.3 a day or two (will be more or less the same thing with a new version number).

**Comment by pietrushnic** [comment]
Thanks. I need it for the conference, so I can't wait.
